arm-notificationhubs/2016-03-01/swagger/notificationhubs.json

```diff
@@ -0,0 +1,1742 @@
+                    "Namespaces"
+                ],
+                "operationId": "Namespaces_DeleteAuthorizationRule",
+                "description": "Deletes a namespace authorization rule",
+                "parameters": [{
+                    "name": "resourceGroupName",
+                    "in": "path",
+                    "required": true,
+                    "type": "string",
+                    "description": "The name of the resource group."
+                }, {
+                    "name": "namespaceName",
+                    "in": "path",
+                    "required": true,
+                    "type": "string",
+                    "description": "The namespace name."
+                }, {
+                    "name": "authorizationRuleName",
+                    "in": "path",
+                    "required": true,
+                    "type": "string",
+                    "description": "Authorization Rule Name."
+                }, {
+                    "$ref": "#/parameters/ApiVersionParameter"
+                }, {
+                    "$ref": "#/parameters/SubscriptionIdParameter"
+                }],
+                "responses": {
+                    "204": {
+                        "description": ""
+                    },
+                    "200": {
+                        "description": ""
+                    }
+                }
+            },
+            "get": {
+                "tags": [
+                    "Namespaces"
+                ],
+                "operationId": "Namespaces_GetAuthorizationRule",
+                "description": "Gets an authorization rule for a namespace by name.",
+                "parameters": [{
+                    "name": "resourceGroupName",
+                    "in": "path",
+                    "required": true,
+                    "type": "string",
+                    "description": "The name of the resource group."
+                }, {
+                    "name": "namespaceName",
+                    "in": "path",
+                    "required": true,
+                    "type": "string",
+                    "description": "The namespace name"
+                }, {
+                    "name": "authorizationRuleName",
+                    "in": "path",
+                    "required": true,
+                    "type": "string",
+                    "description": "Authorization rule name."
+                }, {
+                    "$ref": "#/parameters/ApiVersionParameter"
+                }, {
+                    "$ref": "#/parameters/SubscriptionIdParameter"
+                }],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/SharedAccessAuthorizationRuleResource"
+                        }
+                    }
+                }
+            }
+        },
+        "/{operationStatusLink}": {
```

Comment:
long running operations are handled by the long running extension. Is there a reason to have this operation?